pkg/storaged/storage.scss

```diff
@@ -439,3 +439,12 @@ td.ct-text-align-right {
     // Overlay at 1/3  to mix the light and dark against the trough color
     opacity: 0.333;
 }
+
+$phone: 767px;
```

Comment:
the variabls from pkg/lib/_global-variables.scss should be used here.

Reply:
I got this from pkg/shell/shell.scss, so let's update that as well.

Reply:
Done.  But I think the storage page does not include pkg/lib/__global-variables.scss, since the breakpoint is still at 768px.

Reply:
yeap you can include it with: :@use "_global-variables.scss" as *;

Reply:
storage.scss already has `@import "global-variables";` and your `@use` doesn't seem to have any effect, the breakpoint stays at 768px. So *shrug*, let's leave things as they are.